pixel-point/pixelpoint-website
# feat: new articles

*This PR brings:*
- A Plausible set up article
- Automated web page screenshots article

*Steps to test:*
- Open up deploy preview
- Proceed to [/blog page](https://deploy-preview-59--pixelpoint-website.netlify.app/blog/)
- Check out each of the two featured articles
